### DISCUSSION TITLE
Removing (or not copying) files from (to) wheels

### DISCUSSION THREAD

**Discussion by galleon** [python-poetry/poetry#3826] [Q&A]
Hi All

I am getting better at using poetry but I have still some questions :-)

My [build](https://github.com/galleon/scikit-decide) relies on several native dependencies I have therefore an external `builder.py` that has a custom CMake extension. The binary installs perform well and all `bin`, `share`, `lib` and `include` directories are installed in my tmp wheel directory.

Since I am using orly binaries and shared files I would like to remove the `lib` and `include` directories. I tried (unsuccessfully) to use the exclude keyword giving the destination filenames ...

What is the best way to do that ?

Do we have some kind of post-actions that I could simply use to remove those directories if I cannot prevent them to be created at the install of the dependencies?

Thanks

**Comment by abn** [comment]
Did you try using a [glob](https://github.com/python-poetry/poetry-core/blob/015e05df84cec2a42dfe0701ad83c036a433a474/tests/masonry/builders/fixtures/exclude_nested_data_toml/pyproject.toml#L12)? 

This should mean that the builders will exclude these when making the sdist/bdist. Alternatively, there is nothing stopping you from removing these files as a post-build step in your `builder.py` file.
